src/components/Icons/Download/index.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react';
+
+const Download: React.FC<React.SVGProps<SVGSVGElement>> = ({ fill = 'black', ...props }) => (
+  <svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg" {...props}>
+    <path
+      fillRule="evenodd"
+      clipRule="evenodd"
+      d="M4 12.25C4.41421 12.25 4.75 12.5858 4.75 13V19.25H19.25V13C19.25 12.5858 19.5858 12.25 20 12.25C20.4142 12.25 20.75 12.5858 20.75 13V20C20.75 20.4142 20.4142 20.75 20 20.75H4C3.58579 20.75 3.25 20.4142 3.25 20V13C3.25 12.5858 3.58579 12.25 4 12.25Z"
+      fill={fill}
```

Comment:
you can use `fill = param(fill) #000` so its fill will default to black but can be set via css' fill property